features/validate_image.feature

```diff
@@ -465,6 +465,46 @@ Feature: evaluate enterprise contract
      Then the exit status should be 0
     Then the output should match the snapshot
 
+  Scenario: policy rule filtering on imageUrl
+    Given a key pair named "known"
+    Given an image named "acceptance/ec-happy-day"
+    Given a valid image signature of "acceptance/ec-happy-day" image signed by the "known" key
+    Given a valid Rekor entry for image signature of "acceptance/ec-happy-day"
+    Given a valid attestation of "acceptance/ec-happy-day" signed by the "known" key
+    Given a valid Rekor entry for attestation of "acceptance/ec-happy-day"
+    Given a git repository named "happy-day-policy" with
+      | filtering.rego | examples/filtering.rego |
+    Given policy configuration named "ec-policy" with specification
+    """
+    {
+      "sources": [
+        {
+          "volatileConfig": {
+            "exclude": [
+              {
+                "value": "filtering.always_fail",
```

Comment:
This is not included given the `config` below. Intentional?

Reply:
Would it be simpler to simply include `filtering`?

Reply:
I copied/pasted from another test. Doesn't make sense it's not included though, so fixed both.